src/tools/image_filter_color.py

```diff
@@ -1,12 +1,16 @@
 from enum import Enum
 import numpy as np
 import cv2
+from tools.grayscale import rgb_to_grayscale
+from tools.blur import apply_blur
```

Comment:
What is this function used for?

Reply:
don't use blur function I forgot to remove this.